features/language-plugin.feature

```diff
@@ -323,3 +323,51 @@ Feature: Manage translation files for a WordPress install
       | akismet  | de_DE    | available | installed |
       | jetpack  | de_DE    | none      | installed |
     And STDERR should be empty
+
+
+  @require-wp-4.0
+  Scenario: Install translations for all installed plugin
+    Given a WP install
+    And I run `wp plugin path`
+    And save STDOUT as {PLUGIN_DIR}
+    And an empty {PLUGIN_DIR} directory
+    And I run `wp plugin install akismet --version=4.0 --force`
+    And I run `wp plugin install jetpack --version=6.4 --force`
+
+    When I run `wp language plugin install de_DE`
+    Then the return code should be 1
+    And STDERR should be:
+      """
+      Error: Please specify a plugin, or use --all.
+      """
+    And STDOUT should be empty
```

Comment:
The test is failing but I'm not sure why. Same for the theme command.

Reply:
@ocean90 For a command you expect to fail, you need to use `When I try ...` not `When I run ...`.

Reply:
🤦‍♂️ Thanks!